portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro

```diff
@@ -599,6 +599,13 @@ definition {
 		}
 	}
 
+	@description = "Go to SAML Admin Portlet"
+	macro openSAMLAdmin {
```

Comment:
I suggest renaming this macro to goToSAMLAdmin or to something similar, as [openSAML](https://shibboleth.atlassian.net/wiki/spaces/OS30/overview) is an existing implementation of SAML. This name can lead to the misunderstanding, that we actually use openSAML here.